de.lproj/ShortcutRecorder.strings

```diff
@@ -0,0 +1,36 @@
+/* No comment provided by engineer. */
+"Click to record shortcut" = "Hier klicken, um Tastenkombination aufzuzeichnen";
+
+/* No comment provided by engineer. */
+"Command-" = "%DE string testing%-";
```

Comment:
I changed this translation manually and then pushed it to git to test if it worked, and it looks like it did

Reply:
awesome